core-bundle/assets/controllers/checkbox-controller.js

```diff
@@ -0,0 +1,70 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['checkAll', 'input'];
+
+    static values = {
+        rowGuards: {
+            type: Array,
+            default: ['input', 'label', 'button', 'a', '.operations'],
+        },
+    };
+
+    initialize() {
+        this.start = null;
+    }
+
+    rowClick(event) {
+        if (event.target.closest(this.rowGuardsValue)) {
```

Comment:
according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest), `Element.closest()` expects a string of selectors, not an array.

Reply:
🦆 typing... you are right, this should be changed.

Reply:
See 1381836857b5f7785197d9f40a4350d3fa7712d7

Reply:
any particular reason to use an array instead of just a selector string as the value?

Reply:
See 66c6683450150b08c62aba1209fd18affa1cbbe4